method-proposals/PROPOSAL-did-cheqd.md

```diff
@@ -0,0 +1,142 @@
+* Innovative trust registry / trust infrastructure model natively supported, with hierarchical DID and VC relationships able to be expressed to mirror traditional PKI models that would usually adopt X.509 certificates
+* Support in 5+ Open Source SDKs and libraries
+* Support in both Universal Resolver and Universal Registrar
+
+### Drawbacks
+
+* Token used for underlying transactions ($CHEQ), which may limit adoption owing to lack of regulatory clarity in certain jurisdictions. Although this drawback is mitigated through the support for MiCA regulated stablecoins natively on the network.
+* Volatile transaction pricing for DIDs, based on fluctuations in value of $CHEQ.
+* Immutability of DIDs, which may be problematic if sensistive data such as Personally Identifiable Information (PII) is pointed to within the `serviceEndpoint` or as a DID-Linked Resource.
+
+## Existing materials
+
+- [DID Method Specification](https://docs.cheqd.io/product/architecture/adr-list/adr-001-cheqd-did-method)
+- Implementations
+  - [Veramo SDK plugin](https://docs.cheqd.io/product/sdk/veramo-plugin)
+  - [Credo](https://credo.js.org/guides/getting-started/set-up/cheqd)
+  - [Aries Cloud Agent Python (ACA-Py)](https://docs.cheqd.io/product/sdk/aca-py)
+  - [Dock SDK](https://github.com/docknetwork/sdk)
+  - [Walt.id Community Stack](https://walt.id/blog/p/community-stack)
+  - [cheqd Studio](https://cheqd.io/solutions/cheqd-studio/)
+  - [Universal Registrar Driver](https://github.com/decentralized-identity/universal-registrar)
+  - [Universal Resolver Driver](https://github.com/decentralized-identity/universal-resolver)
+  - [Vidos Universal Resolver](https://vidos.id/docs/services/resolver)
+- Block Explorer (including ability to filter by DID transaction types)
+  - [Mainnet transactions](https://explorer.cheqd.io/transactions)
+  - [Testnet transactions](https://testnet-explorer.cheqd.io/transactions)
+- [cheqd Governance Framework](https://learn.cheqd.io/governance/start)
+- [Documentation and tutorials for use of SDKs or APIs](https://docs.cheqd.io/product)
+- No test suite exists as of Jan 2025
+
+## Meeting the selection criteria
+
+| **Criteria** |
+| -------- |
+| **Alignment with DID Core specification** |
+| Yes, fully compliant with DID specification. |
+| **Security and privacy features** |
+| Yes, the DID Method has security and privacy features primarily established via the resilience and security of the underlying cheqd network blockchain. |
+| **Scalability and performance** |
+| Support for between 5000 and 10,000 transactions per second, with multiple transactions allowed and batched per block |
+| **Ease of implementation and use** |
+| Muiltiple implementation routes, via SDKs and APIs as well as full support in the Universal Registrar and Resolver. Clear documentation and tutorials written for each seperate integration pathway  |
+| **Community adoption and support** |
```

Comment:
Would reference both mainnet and testnet?

Reply:
Done